service/pom.xml

```diff
@@ -47,7 +57,7 @@
         <dependency>
             <groupId>org.opentestsystem.delivery</groupId>
             <artifactId>tds-common-legacy</artifactId>
-            <version>${tds-common-legacy.version}</version>
+            <version>${tds.common.legacy.version}</version>
```

Comment:
Hrm.  I didn't realize that common legacy was being pulled into so many projects.  I didn't realize the checkNotNull was there.